lib/rules/no-useless-call.js

```diff
@@ -71,25 +71,35 @@ function isValidThisArg(expectedThis, thisArg, context) {
 // Rule Definition
 //------------------------------------------------------------------------------
 
-module.exports = function(context) {
-    return {
-        CallExpression: function(node) {
-            if (!isCallOrNonVariadicApply(node)) {
-                return;
-            }
+module.exports = {
+    meta: {
+        docs: {
+            description: "disallow unnecessary calls to `.call()` and `.apply()`",
+            category: "Best Practices",
+            recommended: false
+        },
+
+        schema: []
```

Comment:
Yeah a bit late. We decided on this format a while back and have already converted a lot of rules to this format. So, we need to continue along this path.